api/src/lib/cart/src/delete-cart.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @param {cartPgRepo} cartRepo
+ */
+function DeletecartFactory(deps) {
+    'use strict'
+
+    const { cartRepo } = deps
+
+    const deleteCart = async (ctx) => {
+
+        const logger = ctx.request.state.logger
+        console.log(ctx.state.session.id)
```

Comment:
#picky: why use console.log here? You could just `logger.emit('temp', 'debug', ctx.state.session.id)`